float64 x_world
float64 y_world
float64 z_world
float64 width
float64 length
float64 height
string color
float64 area
float64 confidence
